webpack.config.dev.js

```diff
@@ -63,6 +63,9 @@ module.exports = {
           },
           {
             loader: 'sass-loader',
+            options: {
+              implementation: require('sass'),
```

Comment:
node-sass is deprecated, and the old version that was installed here will not compile on arm64. node-sass recommends using this sass package instead.